### PR TITLE
Fixed zonedTimeToUtc not working with year < 100.

### LINKED ISSUE #87: Years below 100 not allowed?
Hi! Not sure if this is a bug or "intended"/due to some date quirks, but I can't use a year below 100.

```
zonedTimeToUtc('0100-01-01', 'Europe/Berlin').toISOString()
// === "0099-12-31T23:06:32.000Z"

zonedTimeToUtc('0099-01-01', 'Europe/Berlin').toISOString()
// === "-001801-01-01T23:06:32.000Z"
```

### PR DESCRIPTION
**Pull request by make-github-pseudonymous-again**
Fixes #87.

### CODE REVIEW

File: src/zonedTimeToUtc/index.js [hunk 1]
```diff
@@ -36,15 +36,11 @@ export default function zonedTimeToUtc(date, timeZone, options) {
...
-    d.getHours(),
-    d.getMinutes(),
-    d.getSeconds(),
-    d.getMilliseconds()
-  )
+  var tmp = new Date(0)
+  tmp.setUTCFullYear(d.getFullYear(), d.getMonth(), d.getDate())
+  tmp.setUTCHours(d.getHours(), d.getMinutes(), d.getSeconds(), d.getMilliseconds())
+
+  var utc = tmp.getTime()
```
Comment (make-github-pseudonymous-again) [inline src/zonedTimeToUtc/index.js:43]: Refactor using new helper functions in `tzParseTimezone`.

File: src/_lib/tzParseTimezone/index.js [hunk 2]
```diff
@@ -70,23 +70,32 @@ export default function tzParseTimezone(timezoneString, date, isUtcDate) {
...
+
+function DateUTC(fullYear, month, day, hour, minute, second, millisecond) {
+  return newDateDateUTC(fullYear, month, day, hour, minute, second, millisecond).getTime()
+}
+
 function calcOffset(date, timezoneString) {
   var tokens = tzTokenizeDate(date, timezoneString)
 
-  var asUTC = Date.UTC(tokens[0], tokens[1] - 1, tokens[2], tokens[3] % 24, tokens[4], tokens[5])
+  var asUTC = DateUTC(tokens[0], tokens[1] - 1, tokens[2], tokens[3] % 24, tokens[4], tokens[5], 0)
```
Comment (make-github-pseudonymous-again) [inline src/_lib/tzParseTimezone/index.js:98]: It seems milliseconds are dropped here, intentional? I can write a second PR to fix that.

File: src/_lib/tzTokenizeDate/test.js [hunk 3]
```diff
@@ -20,6 +20,11 @@ describe('tzTokenizeDate', function () {
     assert.deepEqual(result, [2020, 1, 23, 0, 0, 0])
   })
 
+  it('works with year < 100', function () {
+    var result = tzTokenizeDate(new Date('0099-01-01T00:00:00.000Z'), 'UTC')
+    assert.deepEqual(result, [99, 1, 1, 0, 0, 0, 0])
```
Comment (make-github-pseudonymous-again) [inline src/_lib/tzTokenizeDate/test.js:25]: Oops the last zero needs to be gone for now, please review changes first, I'll force push later.

**Review by make-github-pseudonymous-again — COMMENTED** [review]
I do not think linting was run on this commit. `eslint` barfs something about a deprecation warning:
```console
(node:620180) [ESLINT_PERSONAL_CONFIG_SUPPRESS] DeprecationWarning: '~/.eslintrc.*' config files have been deprecated. Please remove it or add 'root:true' to the config files in your projects in order to avoid loading '~/.eslintrc.*' accidentally. (found in "../../../.eslintrc")
(Use `node --trace-deprecation ...` to show where the warning was created)
```

Did I do something wrong?

**Comment by marnusw** [comment]
@make-github-pseudonymous-again thank you for working on this. I'd be happy to merge this in once completed and the conflicts resolved.

File: src/_lib/tzTokenizeDate/test.js (hunk 3 above)
Comment (make-github-pseudonymous-again) [reply src/_lib/tzTokenizeDate/test.js]: Done!

**Comment by make-github-pseudonymous-again** [comment]
> @make-github-pseudonymous-again thank you for working on this. I'd be happy to merge this in once completed and the conflicts resolved.

@marnusw I rebased and fixed the test with the extra trailing zero. What about linting? I think the other comments I made can be addressed in a different PR.

**Comment by marnusw** [comment]
Released in `v1.3.2`.

**Pull request merged**